### PR TITLE
PVS-Studio warning fixes in non-module Externals/* (part 3)

### PR DESCRIPTION
**Pull request by dimhotepus**
Part 3 of fixes by PVS-Studio static analyzer tool.

### CODE REVIEW

File: Externals/openal/OpenAL-Windows/Router/alc.cpp [hunk 1]
```diff
@@ -1690,7 +1690,7 @@ ALCAPI ALboolean ALCAPIENTRY alcMakeContextCurrent(ALCcontext* context)
 
                 else
                 {
-                    alCurrentContext->Device->AlcApi.alcMakeContextCurrent(0);
+                    context->Device->AlcApi.alcMakeContextCurrent(0);
```
Comment (dimhotepus) [inline Externals/openal/OpenAL-Windows/Router/alc.cpp:1693]: At least try not to dereference `nullptr` here.

Comment (Xottab-DUTY) [reply Externals/openal/OpenAL-Windows/Router/alc.cpp:1693]: Hmm...

Perhaps, @kcat can correct us if something is wrong here or it's ok? (oh, sorry, I know this code is very old😄)

Comment (Xottab-DUTY) [reply Externals/openal/OpenAL-Windows/Router/alc.cpp:1693]: I think it should be correct.

**Comment by Xottab-DUTY** [comment]
Thank you!

**Pull request merged**